src/main/kotlin/data_structurs/LinkedList.kt

```diff
@@ -0,0 +1,52 @@
+package data_structurs
```

Comment:
Package name misspelled -> should be `data_structures`

Reply:
the package name changed in the next commit